org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/CollectionTestCase.xtend

```diff
@@ -296,4 +314,13 @@ class CollectionTestCase extends AbstractWollokInterpreterTestCase {
 		}'''.interpretPropagatingErrors
 	}
 	
+	@Test
+	def void sumNoArgs() {
+		'''
+		program p {
+			«instantiateCollectionAsNumbersVariable»
+			assert.equals(34, numbers.sum())
```

Comment:
@matifreyre  Could we add tests for the edge case of empty and unitary collections?